android/assets/jsons/translations/German.properties

```diff
@@ -5835,11 +5835,10 @@ The Maya measured time in days from what we would call 11th of August, 3114 BCE.
 Unciv only displays ය B'ak'tuns, ඹ K'atuns and ම Tuns (from left to right) since that is enough to approximate gregorian calendar years. The Maya numerals are pretty obvious to understand. Have fun deciphering them! = Unciv zeigt nur ය B'ak'tuns, ඹ K'atuns und ම Tuns (von links nach rechts) an, da dies ausreicht, um gregorianische Kalenderjahre anzunähern. Die Maya-Ziffern sind ziemlich einfach zu verstehen. Viel Spaß beim Entschlüsseln!
 
  # Requires translation!
-Your cities will periodically demand different luxury goods to satisfy their desire for new things in life. = 
+Your cities will periodically demand different luxury goods to satisfy their desire for new things in life. = Eure Städte werden wiederholend verschiedene Luxusgüter fordern, welche deren Verlangen nach neuen Dingen erfüllen.
  # Requires translation!
-If you manage to acquire the demanded luxury by trade, expansion, or conquest, the city will celebrate We Love The King Day for 20 turns. = 
+If you manage to acquire the demanded luxury by trade, expansion, or conquest, the city will celebrate We Love The King Day for 20 turns. = Wenn Sie das geforderte Luxusgut durch Handel, Grenzerweiterung, oder Eroberung erhalten, wird die Stadt den Wir Lieben den König Tag für 20 Runden feiern.
  # Requires translation!
-During the We Love The King Day, the city will grow 25% faster. = 
+During the We Love The King Day, the city will grow 25% faster. = Während des Wir lieben den König Tag, wächst ihre Stadt um 25% schneller.
  # Requires translation!
-This means exploration and trade is important to grow your cities! = 
-
+This means exploration and trade is important to grow your cities! = Das bedeutet, dass Erkunden und Handel betreiben einen wichtigen Beitrag dazu leisten Ihre Städte wachsen zu lassen!
```

Comment:
"Das bedeutet, dass Erkundung und Handel wichtig sind, um deine Städte wachsen zu lassen!" liest sich in meinen Augen flüssiger.